packages/components/organisms/f-takeawaypay-activation/src/components/TakeawaypayActivation.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div
+        :class="$style['c-takeawaypayActivation']"
+        data-test-id="takeawaypayActivation">
+        {{ copy.text }}
+    </div>
+</template>
+
+<script>
+import { globalisationServices } from '@justeat/f-services';
+import tenantConfigs from '../tenants';
+
+export default {
+    name: 'TakeawaypayActivation',
```

Comment:
You don't need to set this property as vue-cli will add it automatically for you based on the file name.

Reply:
Sure, will correct. Was just auto generated code

Reply:
Sorry about that, we noticed it the other day :)